docker/use

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+set -ex
+HERE=$(dirname $0)
+. $HERE/common
+
+NETWORK=beebop_nw
+VOLUME=beebop-storage
+NAME_REDIS=beebop-redis
+NAME_API=beebop-py-api
+NAME_WORKER=beebop-py-worker
+
+docker volume create $VOLUME
+docker run --rm -v $VOLUME:/beebop/storage $TAG_SHA \
+       ./scripts/download_db --small storage
+docker network create $NETWORK > /dev/null || /bin/true
+
+docker run -d --rm --name $NAME_REDIS --network=$NETWORK redis:5.0
+docker run -d --rm --name $NAME_WORKER --network=$NETWORK \
+       --env=REDIS_URL="redis://$NAME_REDIS:6379" \
+       -v $VOLUME:/beebop/storage \
+       $TAG_SHA rqworker
+docker run -d --rm --name $NAME_API --network=$NETWORK \
+       --env=REDIS_URL="redis://$NAME_REDIS:6379" \
+       -v $VOLUME:/beebop/storage \
+       -p 8080:8080 \
```

Comment:
I have previously used port 5000 to access the api, since vue also uses 8080 as default. Are there any conventions regarding port numbers?
```suggestion
       -p 5000:8080 \
```

Reply:
No, anything above 1024 is fair game - see https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers - it's best to stay away from ones that have a well known meaning.

Remapping will *probably* work but it's best to make the change the whole way through to use 5000 for the flask API